src/main/java/frc/trigon/robot/subsystems/leds/LedStrip.java

```diff
@@ -0,0 +1,108 @@
+package frc.trigon.robot.subsystems.leds;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.Robot;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+public class LedStrip extends SubsystemBase {
```

Comment:
add function that turns off the strip and use the logic of the end function:

    Color[] colors = new Color[ledStrip.getLength()];
    for (int i = 0; i < ledStrip.getLength(); i++)
        colors[i] = Color.kBlack;
    ledStrip.setLedsColors(colors);

and in your end function call the funciton that you would add to the subsystem.

something like:

    void turnLedsOff() {
        ...
    }